<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calificacion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Asistencia</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_grade">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_attendance">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar Estudiante</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar reporte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Estudiante</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblGrades">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_students">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista de estudiantes</string>
   </property>
  </widget>

  <!-- NUEVOS FILTROS -->
  <widget class="QSpinBox" name="filter_min_grade">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>610</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="filter_attendance">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>610</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filter_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>610</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
